system_baseline/mgmt_views/v0.py

```diff
@@ -9,7 +9,7 @@
 
 RANGES = text(
     "select count(*) from"
-    " (select count(account) from system_baselines group by account) x"
+    " (select count(org_id) from system_baselines group by org_id) x"
```

Comment:
Do we need an if/else for which to show here as well?

Or is this also awaiting a final answer/mock from ux here:
https://issues.redhat.com/browse/RHCLOUD-18660?focusedCommentId=20610600&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-20610600

Reply:
To be honest I don't know. This is used for reporting only. I changed queries to use `org_id` instead of `account` in this PR.

I can revert it back now, but the question is still here.... maybe @jeromemarc can answer this?

Reply:
Yeah, I'm not sure if we need both for reporting for now or not.